scripts/e2e.min.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# -------------------------------------------------------------------------------------
+# Runs basic min bundle loading check on chrome headless
+# -------------------------------------------------------------------------------------
+
+# Exit immediately on error
```

Comment:
Which version of chrome will be used? It could be important to not use the latest version which probably does support some features older supported browsers don't have.

Reply:
Yes this is possible, I'm not sure. We use whatever version is installed in Travis. I think you asked me to do this with #3148 because you didn't like the chromium download wait.

Reply:
Yes, I thought our ``e2e`` tests could use the locally installed chrome bin if existing on Travis or from our PC's to speed up the installation process of web3.js. I think this is something we could also address in another separate PR because it would affect other parts of our QA as well. 

Let us discuss this closer on the next call or in an issue here on GitHub.